.travis.yml

```diff
@@ -1,8 +1,6 @@
 os: linux
-dist: bionic
```

Comment:
`dist` was specified 2 times (which should be disabled by YAML but Travis seems to be gentle on this), use only the most recent distribution instead.